Sources/SpeziFirebaseAccountStorage/SpeziFirebaseAccountStorage.docc/SpeziFirebaseAccountStorage.md

```diff
@@ -36,15 +36,36 @@ import SpeziFirebaseAccount
 import SpeziFirebaseAccountStorage
 
 class ExampleAppDelegate: SpeziAppDelegate {
-override var configuration: Configuration {
-    Configuration {
-        AccountConfiguration(
-            service: FirebaseAccountService(),
-            storageProvider: FirestoreAccountStorage(storeIn: Firestore.firestore().collection("users"))
-            configuration: [/* ... */]
-        )
+    override var configuration: Configuration {
+        Configuration {
+            AccountConfiguration(
+                service: FirebaseAccountService(),
+                storageProvider: FirestoreAccountStorage(storeIn: Firestore.firestore().collection("users")),
+                configuration: [/* ... */]
+            )
+        }
     }
 }
+
+### Custom Encoder/Decoder Configuration
+
+For advanced use cases, such as integrating with libraries like PhoneNumberKit where you might want to set specific encoding/decoding strategies,
+you can provide custom encoder and decoder instances with specific userInfo configurations.
+
+```swift
+import PhoneNumberKit
+
+let encoder = FirebaseFirestore.Firestore.Encoder()
+encoder.userInfo[CodingUserInfoKey(rawValue: "com.roymarmelstein.PhoneNumberKit.encoding-strategy")!] = PhoneNumberEncodingStrategy.e164
```

Comment:
@PSchmiedmayer since the CodingKeys in PhoneNumberKit are internal, I used the rawValue to access it.

Reply:
Good catch; took a quick look at the package and added a small PR that would address this: https://github.com/marmelroy/PhoneNumberKit/pull/856. We can probably anticipate that this can be merged in the near future.